index.js

```diff
@@ -0,0 +1,68 @@
+// EXERCISE 1
+console.log(document.getElementById('mainTitle'));
+
+// EXERCISE 2
+document.getElementById('content').innerHTML = '<p>Updated Content</p>';
+
+// EXERCISE 3
+document.getElementById('content').style.backgroundColor = 'lightgreen';
+
+// EXERCISE 4
+document.getElementById('list').classList.add('active');
+
+setTimeout(() => {
+  document.getElementById('list').classList.remove('active');
+}, 2000);
+
+// EXERCISE 5
+let newButton = document.createElement('button');
+
+newButton.textContent = 'Click Me';
+document.getElementById('content').appendChild(newButton);
+
+// EXERCISE 6
+let resultElement = document.getElementById('result');
+
+if (resultElement) {
```

Comment:
Why do you add a conditional statement? 🤔

Reply:
Fixed.